packages/stems/src/components/PillButton/types.ts

```diff
@@ -0,0 +1,18 @@
+import { HTMLAttributes } from 'react'
+// import { HTMLAttributes, ReactNode } from 'react'
+import * as React from 'react'
+
+export enum Type {
+  PRIMARY = 'primary',
+  SECONDARY = 'secondary'
+}
+
+export type PillButtonProps = HTMLAttributes<HTMLButtonElement> & {
```

Comment:
Can we extend on Button, so 
```typescript
type PillButtonProps = ButtonProps
```
or if we are gonna do the variant change (which now makes me think lets not do it and just do it with button eventually:

```typescript
type PillButtonProps = ButtonProps & {
  variant: PillVariant
}
```